docs/migrations/v7-to-v7_1.md

```diff
@@ -1,19 +1,46 @@
 # Migrating from v7 to v7.1
 
-This guide provides instructions for migrating to a new version of ibc-go.
+This guide provides instructions for migrating to version `v7.1.0` of ibc-go.
 
 There are four sections based on the four potential user groups of this document:
 
-- [Chains](#chains)
-- [IBC Apps](#ibc-apps)
-- [Relayers](#relayers)
-- [IBC Light Clients](#ibc-light-clients)
+- [Migrating from v7 to v7.1](#migrating-from-v7-to-v71)
+  - [Chains](#chains)
+  - [IBC Apps](#ibc-apps)
+  - [Relayers](#relayers)
+  - [IBC Light Clients](#ibc-light-clients)
 
 **Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated on major version releases.
 
 ## Chains
 
-- No relevant changes were made in this release.
+In the previous release of ibc-go, the localhost `v1` light client module was deprecated and removed. The ibc-go `v7.1.0` release introduces `v2` of the 09-localhost light client module.
+
+<!-- TODO: Update the links to use release version instead of feat branch -->
+An [automatic migration handler](https://github.com/cosmos/ibc-go/blob/09-localhost/modules/core/module.go#L133-L145) is configured in the core IBC module to set the localhost `ClientState` and sentintel `ConnectionEnd` in state.
+
+In order to use the 09-localhost client chains must update the `AllowedClients` parameter in the 02-client submodule of core IBC. This can be configured directly in the application upgrade handler or alternatively updated via the legacy governance parameter change proposal.
```

Comment:
I checked with @womensrights and she is fine with this approach, but she suggested to add a line here advising/encouraging chains to enable it. So maybe something like "We **strongly** recommend chains to perform this action".